devtools/src/shared/rpc/custom-values.ts

```diff
@@ -1,7 +1,7 @@
 export interface NonSerializableValue_Base {
   __custom: '@@pc-non-serializable'
   __type: string
-  value: string
+  value: string | number | boolean | null | object
```

Comment:
Wouldn't using `unknown` here make more sense? And then we can specify the actual value everywhere, it's a bit strictker